PocketKit/Sources/Sync/V3/V3RequestTypes.swift

```diff
@@ -0,0 +1,32 @@
+protocol BasicV3Request {
```

Comment:
The Sync package _does_ have a `Session` protocol (see `ApolloClient+Extensions.swift` and its usage in `AppSession+Sync.swift`). There might be an opportunity for extension (if / as necessary) and reuse. Maybe even just having this request have a `var session: Session? { get }` requirement rather than explicitly defining other values? Not sure, just an idea.